ts_sdk/task/__util_log.py

```diff
@@ -18,30 +18,39 @@
 error_type_key = "errorType"
 exception_key = "exception"
 
+
 class Log:
     def __init__(self, context):
         self._context = context
-    
-    def log(self, *args):        
+
+    def log(self, *args):
         for arg in args:
-          try:
-            # Perform actual logging, that will be transported to Cloudwatch.
-            __builtins__['default_print'](json.dumps(self.create_json_log_entry(arg)))
-          except:
-            __builtins__['default_print'](arg)
-    
+            try:
+                # Perform actual logging, that will be transported to Cloudwatch.
+                __builtins__["default_print"](
+                    json.dumps(self.create_json_log_entry(arg))
+                )
+            except TypeError:
+                __builtins__["default_print"](
+                    TypeError("Object of type type is not JSON serializable")
+                )
+                __builtins__["default_print"](arg)
```

Comment:
Are you sure the object is always of type `type`?  I think this might be safer:
```suggestion
            except TypeError as ex:
                __builtins__["default_print"](repr(ex))
                __builtins__["default_print"](arg)
```